examples/ui/resources/display.ron

```diff
@@ -6,5 +6,5 @@
   multisampling: 0,
   title: "UI example",
   visibility: true,
-  vsync: true,
+  vsync: false,
```

Comment:
Probably shouldn't leave this

Reply:
It was to see the framerate drops caused by gfx-glyph when changing a text dimension. I can put it back to true.

Reply:
fixed